vtex-apps/checkout-graphql
# CHK-479: Add support for sales channel in addToCart mutation

#### What problem is this solving?

Adds support for the `salesChannel` parameter in the `addToCart` mutation.

#### How should this be manually tested?

[Workspace](https://saleschannel--checkoutio.myvtex.com/cart/add?sku=312&sc=2).

Notice the `sc=2` querystring in the add-to-cart URL above.

#### Checklist/Reminders


#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements
